tests/test_lookml.py

```diff
@@ -1832,29 +1832,34 @@ def test_context_id(runner, glean_apps, tmp_path):
         )
 
         expected_explore = {
-            "includes": ["/looker-hub/custom/views/context.view.lkml"],
             "explores": [
                 {
-                    "sql_always_where": "${context.submission_date} >= '2010-01-01'",
-                    "view_name": "context",
                     "always_filter": {
                         "filters__all": [[{"submission_date": "28 days"}]]
                     },
                     "joins": [
                         {
-                            "relationship": "one_to_many",
-                            "sql": "LEFT JOIN UNNEST(${context.contexts}) AS context__contexts",
                             "name": "context__contexts",
+                            "relationship": "one_to_many",
+                            "sql": "LEFT JOIN UNNEST(${context.contexts}) "
+                            "AS context__contexts",
```

Comment:
I think I preferred the old formatting, easier to read.

```suggestion
                            "sql": "LEFT JOIN UNNEST(${context.contexts}) AS context__contexts",
```